.devcontainer/Dockerfile

```
@@ -0,0 +1,46 @@
+ARG VARIANT=3.8
+FROM --platform=linux/amd64 mcr.microsoft.com/vscode/devcontainers/python:${VARIANT}
+
+# System update/upgrade
+RUN apt-get update
+RUN apt-get -y upgrade
```

```suggestion
RUN apt-get update && apt-get -y upgrade
```
Let's run the upgrade only if the update was successfull !